SU-SWS/stanford_components
# EARTH-280: Left center right variant options for feature blocks

# READY FOR REVIEW

# Summary
- Adds new variants to section feature blocks paragraph type

# Needed By (Date)
- Sometime before end of the sprint so I can post it for Nick to review.

# Steps to Test

1. Check out this branch and EARTH-280 in stanford_paragraph_types
2. Revert `drush features-import stanford_paragraph_feat_blocks -y`
3. Edit the home page and choose a few variants from the feature blocks p-type
4. Review front end for placement

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
